canopen/sdo/base.py

```diff
@@ -80,14 +80,14 @@ def download(
         raise NotImplementedError()
 
 
-class Record(Mapping):
+class SdoRecord(Mapping):
```

Comment:
Same here, `SDORecord`?  Please just be consistent.

Reply:
With reference to comment above, `SdoRecord` is correct if `SdoBase` is correct I would argue.